AdminCore.Services/ScheduleService.cs

```diff
@@ -0,0 +1,46 @@
+﻿using AdminCore.Common.Interfaces;
+using AdminCore.DAL;
+using AdminCore.DAL.Models;
+using AdminCore.DTOs;
+using AdminCore.Services.Base;
+using AutoMapper;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace AdminCore.Services
+{
+  public class SchedulesService : BaseService, ISchedulesService
+  {
+    private readonly IMapper _mapper;
+    private readonly IScheduleRunner _scheduleRunner;
+
+    public SchedulesService(IDatabaseContext database, IMapper mapper, IScheduleRunner scheduleRunner)
+                : base(database)
+    {
+      _mapper = mapper;
+      _scheduleRunner = scheduleRunner;
+    }
+
+    public IList<ScheduleDto> GetAllSchedules(bool? active)
+    {
+      var schedules = DatabaseContext.SchedulesRepository.Get(x => x.IsActive).ToList();
```

Comment:
You probably want to use the active parameter here (also handle when there isn't one.)